matplotcheck/tests/test_base_data.py

```diff
@@ -56,6 +58,15 @@ def test_assert_xydata_changed_data_points_only(pt_scatter_plt, pd_df):
     plt.close()
 
 
+def test_assert_xydata_floatingpoint_error(pt_scatter_plt, pd_df):
+    for i in range(len(pd_df["A"])):
+        pd_df["A"][i] = pd_df["A"][i] + 1.0e-10
+    pt_scatter_plt.assert_xydata(pd_df, xcol="A", ycol="B", points_only=True)
+
+
+""" LABELS DATA TESTS """
+
+
 def test_assert_xydata_xlabel(pt_bar_plt, pd_df):
     "Tests the xlabels flag on xydata"
```

Comment:
Docstrings should have 3 `"""` rather than 1. please fix this throughout! i noticed it several times.